org.gnome.Maps.json

```diff
@@ -149,8 +151,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/libshumate/1.1/libshumate-1.1.0.tar.xz",
-                    "sha256": "b3b7c4c2d3002f37b951c9ca99b09275d0aa82265c69f53374ce12faefd3b14e",
+                    "url": "https://download.gnome.org/sources/libshumate/1.0/libshumate-1.0.5.tar.xz",
+                    "sha256": "a8cfd8df9177ee44b04733b017af13c9e93f98765cdc89b750a48edaf701956c",
                     "x-checker-data": {
                         "type": "gnome",
                         "stable-only": true,
```

Comment:
This line is used to signal that it won't accept odd numbered releases as stable, unless the the version number is above 40, for example 45.X.Y. I would remove it.

Reply:
If I am not mistaken it also knows that e.g. 46.beta is unstable.

Reply:
Yeah, that seems off here…

Reply:
Its just a hack I implemented to try to understand the GNOME versioning scheme, but libs that dont use shouldnt use it.